src/applications/edu-benefits/complaint-tool/config/form.js

```diff
@@ -287,6 +305,93 @@ const formConfig = {
         }
       }
     },
+    schoolInformation: {
+      title: 'School Information',
+      pages: {
+        schoolInformation: {
+          path: 'school-information',
+          title: 'School Information',
+          uiSchema: {
+            school: {
+              facilityCode: {
+                'ui:title': 'Please click on the button to search for your school.',
+                'ui:field': SchoolSelectField,
+                'ui:options': {
+                  hideIf: formData => formData.school['view:cannotFindSchool']
+                }
+              },
+              'view:manualSchoolEntry': {
+                name: {
+                  'ui:title': 'Name',
+                  'ui:required': formData => _.get('school.view:cannotFindSchool', formData)
```

Comment:
Tried a couple of different formats for `oneOf` in the schema but I couldn't get this to be automatically enforced with just schema settings.

Reply:
We should update the documentation on this.

Reply:
The wiki? I'm never sure what documentation to update on the vets-team repo.